nemo/collections/asr/models/label_models.py

```diff
@@ -124,21 +122,6 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             weight = [sum(self.labels_occurrence) / (len(self.labels_occurrence) * i) for i in self.labels_occurrence]
 
         if 'loss' in cfg:
-            # To support older version checkpoints
```

Comment:
how about older checkpoints such as speakernet?

Reply:
Latest versions are being published on NGC. Urls would be updated accordingly

Reply:
Then let's merge and update the link until the latest version is published? Or it will break things.

Reply:
Added links for speakernet and ecapa models as well. All models are updated now